File: C_Assignment/Assignment_40/programAQ2.c
///////////////////////////////////////////////////////////
//
//  File name : Assignement40Question2.c
//  Description : Write a recursive program which display below pattern
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Output : 1 2 3 4 5
Prototype :

*/

#include<stdio.h>
void Display()
{
    static int i = 1;

    if(i <= 5)
    {
        printf("%d\t",i);
        i++;
        Display();
    }
}

int main()
{
   Display();

    return 0;
}

File: Java_Assignment/Assignment_48/programAQ2.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement48Question2.java
//  Description : Write Java program which accept String from user and display
//                 below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*
Input : Hello
Output : 
H e l l o
H e l l
H e l
H e
H
Program Layout :

*/

import java.util.Scanner;

class Pattern
{
    public void PatternX(String str)
    {
        int i = 0,j = 0;

        for(i = str.length();i >= 1;i--)
        {        
            for(j = 0;j < i;j++)
            {
                System.out.print(str.charAt(j) + "\t");
            }
            System.out.println();
        }
    }
}

public class programAQ2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        String str = "";

        System.out.println("Enter String: ");
        str = sobj.nextLine();

        Pattern pobj = new Pattern();

        pobj.PatternX(str);

        sobj.close();
    }   
}